docs/user/TerraformAzureAuthentication.md

```diff
@@ -21,13 +21,11 @@ To authenticate Terraform to access Azure, you will need to set the following fo
 | client_id | your app_id when using a Service Principal | string | "" |
 | client_secret | your client secret when using a Service Principal| string | ""|
 
-TF_VAR_name environment variables for these would be
+Once authenticated to the `az cli`, the following [bash (code) script](../../files/TerraformEnvVariableAssignment.sh) can be used to (re)assign the TF Environment Variables:
 
-```
```

Comment:
Please add these variables back in. The bash script an an optional piece. We should not remove telling people exactly what they can set.